controllers/user.js

```diff
@@ -0,0 +1,15 @@
+const User = require('../models/user');
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Nlopez4/fitness_tracking_app/pullRequest?prid=7812747)
```suggestion
const User = require("../models/user");
```